R/utilities-pk-analysis.R

```diff
@@ -95,5 +95,16 @@ pkAnalysesToDataFrame <- function(pkAnalyses) {
     }
   )
 
-  return(dplyr::as_tibble(dataFrame))
+  # consistently return a (classical) data frame
+  return(as.data.frame(df))
```

Comment:
Same question as for  `dataSetToDataFrame`.

Reply:
Reverted